src/main/java/org/opensrp/web/config/SentryConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package org.opensrp.web.config;
+
+import io.sentry.Sentry;
+import io.sentry.spring.EnableSentry;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+
+import javax.annotation.PostConstruct;
+
+@EnableSentry
+@Configuration
+public class SentryConfiguration {
+
+	@Value("${sentry.dsn}")
```

Comment:
add default of empty in case value is not provided. also alias with the config file that has the variables e.g 
`#{opensrp['sentry.dsn'] ?: ""}`